### ISSUE TITLE
Unknown function: chansend

### ISSUE THREAD

**Issue by 3N4N** [ojroques/vim-oscyank#44]
https://github.com/ojroques/vim-oscyank/blob/0e6543a2a3ff009382cb04f30c7ccbe421a6dc72/plugin/oscyank.vim#L118

This is available only in neovim.  It should have an alt for Vim as well.

**Comment by ojroques** [comment]
Why your system is using the fallback? What is your OS?

**Comment by 3N4N** [comment]
Windows.

**Comment by ojroques** [comment]
I've added a new fallback. I'm not using Windows though, let me know if it works that way.

**Comment by 3N4N** [comment]
Yes, it does.  I came up with the same solution (when I say "came up," I mean I stole it from [hterm][1]).  But it has an issue: the screen flashes every time you redraw.  That's not idea.

What I ended up doing is using win32 clipboard implementation of Vim when on Windows.  Cause it's guaranteed to work.  And if I ssh into a Linux machine, Vim will use this plugin so it doesn't have rely on X11.

[1]: https://chromium.googlesource.com/apps/libapps/+/master/hterm/etc/osc52.vim
